src/raylib.cc

```diff
@@ -740,6 +748,24 @@ static PyObject* export_function(PyObject* self, PyObject* args) {
   }
 }
 
+static PyObject* export_reusable_variable(PyObject* self, PyObject* args) {
+  Worker* worker;
+  const char* name;
+  int name_size;
+  const char* initializer;
+  int initializer_size;
+  const char* reinitializer;
+  int reinitializer_size;
+  if (!PyArg_ParseTuple(args, "O&s#s#s#", &PyObjectToWorker, &worker, &name, &name_size, &initializer, &initializer_size, &reinitializer, &reinitializer_size)) {
+    return NULL;
+  }
+  std::string name_str = std::string(name, static_cast<size_t>(name_size));
+  std::string initializer_str = std::string(initializer, static_cast<size_t>(initializer_size));
+  std::string reinitializer_str = std::string(reinitializer, static_cast<size_t>(reinitializer_size));
```

Comment:
Nit: `std::string name_str(name, static_cast<size_t>(name_size))` is the way to do this.